features/sign_in_and_sign_up.feature

```diff
@@ -1,30 +1,30 @@
-Feature: User login and registration
+Feature: Sign in and Sign up
```

Comment:
I noticed we were switching between terms in the code and the features, and it was throwing me off a bit. I decided to side with Devise and use the same terms they use to prevent the switching cost. I can pull this change out pretty easily if we decide we want it to be independent or not at all.

Reply:
💯 anti-confusion measure